styles.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: purple;
+}
```

Comment:
- [ ] [optional] It would have been nice to use classes to style your header. This is recommended as it will help you to be able to reuse the same code for multiple elements that share the same style, avoiding reputation and saving you time.